CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## [3.6.0](https://github.com/marklogic-community/mlxprs/releases/tag/v3.6.0)
+
+- [#116](https://github.com/marklogic-community/mlxprs/issues/116) Updated dependencies and fixed all known security vulnerabilities
```

Comment:
You don't need a link - just do "#116" and GitHub will automatically create a link for you.